devbox.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "https://raw.githubusercontent.com/jetify-com/devbox/0.13.7/.schema/devbox.schema.json",
+  "packages": {
+    "gcc":           "14.2",
+    "pkg-config":    "0.29.2",
+    "icdiff":        "latest",
+    "stack":         "latest",
+    "cabal-install": "latest",
+    "zlib": {
+      "version": "latest",
+      "outputs": ["out", "dev"]
+    },
+    "hpack": "latest"
+  },
+  "env": {
+    "LD_LIBRARY_PATH": "$DEVBOX_PACKAGES_DIR/lib:$LD_LIBRARY_PATH"
+  },
```

Comment:
It seems important that the `zlib` dependency has the `outputs` field, and that we export the `LD_LIBRARY_PATH` env variable. Otherwise, we would have issues with `pkg-config` finding the `zlib` library during compilation.

Reply:
Huh, nice. I've had issues with zlib paths before – I guess this fixes that!